velox/exec/fuzzer/JoinFuzzer.cpp

```diff
@@ -510,6 +510,21 @@ core::PlanNodePtr tryFlipJoinSides(const core::HashJoinNode& joinNode) {
       joinNode.outputType());
 }
 
+core::PlanNodePtr tryFlipJoinSides(const core::NestedLoopJoinNode& joinNode) {
```

Comment:
Hi @xiaoxmeng, I actually intentionally make the methods use the same name for both nested loop join and hash join. This is because I call tryFlipJoinSides() in addFlippedJoinPlan<TNode>() that works for both join nodes.